hvpy/tests/test_datasources.py

```diff
@@ -0,0 +1,24 @@
+from hvpy.datasources import DataSources
+
+
+def test_datasources():
```

Comment:
Add a test where you query getDataSources endpoint and confirm that each ID returned is in the enum.